### PR TITLE
LOCUS-42 - add jsonschema2pojo lib

### PR DESCRIPTION
**Pull request by pleacu**
Add jsonschema2pojo and dependent locus target platform jars.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,10 +14,11 @@
     <targetEclipseVersion>4.3 (Kepler)</targetEclipseVersion>
     <targetEclipseVersionName>juno</targetEclipseVersionName>
     <update.site.description>&amp;shy;</update.site.description>
-    <TARGET_PLATFORM_VERSION>1.0.0.Final</TARGET_PLATFORM_VERSION>
+    <TARGET_PLATFORM_VERSION>1.3.0.Final</TARGET_PLATFORM_VERSION>
     <tycho.scmUrl>scm:git:https://github.com/jbosstools/jbosstools-locus.git</tycho.scmUrl>
     <previousVersion>1.2.0.Final</previousVersion>
     <baseline>http://repository.jboss.org/nexus/content/unzip/unzip/org/jboss/tools/locus/update.site/${previousVersion}/update.site-${previousVersion}.zip-unzip/</baseline>
+<!--    <targetplatform.url>file://${project.build.directory}/target-platforms/multiple/target/locus-multiple.target.repo/</targetplatform.url> -->
```
Comment (mickaelistria) [inline pom.xml]: Are this comment useful? If not, please consider removing them.

File: pom.xml [hunk 2]
```diff
@@ -325,6 +326,20 @@
...
+      <url>${targetplatform.url}</url>
+      <layout>p2</layout>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+    </repository>
+-->
```
Comment (mickaelistria) [inline pom.xml]: Are this comment useful? If not, please consider removing them.

**Comment by mickaelistria** [comment]
Overall, that seems pretty good. I'm only wondering what is the value of the commented parts?

**Comment by pleacu** [comment]
Apologies - identified commentary was experimental - it has been removed.

**Comment by pleacu** [comment]
@mickaelistria @nickboldt  - got a +1 from @maxandersen for this PR.  Please merge at your convenience

**Comment by nickboldt** [comment]
Well, that was convenient. :D

**Pull request merged**